doc/markdownlinkconverter/README.md

```diff
@@ -28,11 +28,26 @@ happens in 2 passes, which is needed because there can be files with no title.
 
 ## Link Validation
 
+### internal links
+
 The link validation works with a simple try to `fopen` the file,
 which the link refers to.
 
+### external links
+
+Every link starting with `http`, `https` or `ftp` will be written to a file named `httpLinks.txt` located in your
+build folder. With the following syntax:
+
+	<file>|<line> col 0 | <url>
+
+Note: Due to the nature of the Markdown Link Converter the file can only be opened in append mode. So delete it and rerun the
+html build process (`make clean` could be needed) to get a list without duplicates.
+
+In the script folder is a script named `httpLinksChecker`. This script can be used to validate the links.
```

Comment:
Please use `-` names for scripts, e.g. `link-checker` or `external-link-checker`.